src/vanta_ledger/routes/users.py

```
@@ -0,0 +1,9 @@
+from fastapi import APIRouter, Depends
+from typing import List
+from ..auth import AuthService
+
+router = APIRouter(prefix='/users', tags=['Users'])
+
+@router.get('/')
+async def get_users(current_user: dict = Depends(AuthService.verify_token)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)